### ISSUE TITLE
Eating Follower Heal

### ISSUE THREAD

**Issue by facheme** [L2J/L2J_DataPack#269]
The Cannibalistic Stakato Leader has a skill (id 4484) that regenerates half his HP without reuseDelay.

**Comment by Zoey76** [comment]
Yeah, this was reported by @u3games here http://www.l2jserver.com/forum/viewtopic.php?f=103&t=26415
